sympy/polys/ring_series.py

```diff
@@ -101,18 +93,19 @@ def rs_is_puiseux(p, x):
     >>> rs_is_puiseux(p, x)
     True
     """
-    ii = p.ring.gens.index(x)
-    n = 1
+    index = p.ring.gens.index(x)
     for k in p:
-        if k[ii] != int(k[ii]):
+        if k[index] != int(k[index]):
             return True
-        if k[ii] < 0:
+        if k[index] < 0:
```

Comment:
Setting up the PR so that it is two commits (one for PEP8, one for semantics) is enough, because people are visiting the commit log and are not overly concerned with PRs. Just make sure that each commit has the appropriate comment.